portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -396,8 +397,9 @@ async function loadChartData() {
   return chartData;
 }
 
-google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawInteractiveChart);
```

Comment:
does the chart still operate seemlessly with these variables removed?

Reply:
The variables are currently commented out because there was a conflict between the chart and typewriter feature, both won't work seemlessly at the same time. The added TODO is to fix this problem in another PR.